src/locales/en/constructor.json

```diff
@@ -0,0 +1,19 @@
+{
+  "titles": {
+    "model": "Model",
+    "basis": "Basis",
+    "pattern": "Pattern",
+    "bottom": "Bottom",
```

Comment:
Some translations have a common use, move it to some general translation file like common.json, discuss it with your teammates they have the same comments.

Reply:
Words like Size, Bottom, Model, Submit, Cancel, look through your translations. Thanks,